weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/SchemaMappingStep.tsx

```diff
@@ -0,0 +1,352 @@
+import {Box, Paper, Stack, Tooltip, Typography} from '@mui/material';
+import React, {useEffect, useState} from 'react';
+
+import {parseRef} from '../../../../../react';
+import {Select} from '../../../../Form/Select';
+import {Icon} from '../../../../Icon';
+import {LoadingDots} from '../../../../LoadingDots';
+import {Pill} from '../../../../Tag/Pill';
+import {CopyableId} from '../pages/common/Id';
+import {useWFHooks} from '../pages/wfReactInterface/context';
```

Comment:
I think this page is a bit confusing for a new-comer. I would consider some help text or info section that helps them understand the what they are doing with this mapping